packages/legend-query-builder/src/components/QueryBuilderResultPanel.tsx

```diff
@@ -272,8 +272,30 @@ const QueryBuilderGridResultContextMenu = observer(
           );
     };
 
+    const handleCopyCellValue = applicationStore.guardUnhandledError(() =>
+      applicationStore.copyTextToClipboard(event?.value),
+    );
+
+    const handleCopyRowValue = applicationStore.guardUnhandledError(() =>
+      applicationStore.copyTextToClipboard(JSON.stringify(event?.data)),
+    );
+
     return (
       <MenuContent ref={ref}>
+        <MenuContentItem
```

Comment:
Let's move these generic actions to the bottom and add a `MenuDivider` between them